pkg/generate/code/common.go

```diff
@@ -62,7 +67,6 @@ func FindIdentifiersInCRD(
 		r.Names.Original + "Id",
 		r.Names.Original + "Ids",
```

Comment:
I just noticed this... we should move the `r.Names.Original + "Id[s]"` lines above the `"Name"` line.